README.md

```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
 
 ## Workflow
 * So you want to contribute to cohort resources? you've come to the right spot
+### Jira Workflow 
+* [Jira Link](https://appacademyio.atlassian.net/secure/RapidBoard.jspa?rapidView=86&projectKey=IN&selectedIssue=IN-91)
+* This is a slightly modified outline of the workflow for the curriculumn team
+#### submitting a ticket
+* Submitting a ticket is quite easy. First create a new task on the Curriculum Issues Backlog:
+    ![label picture jira](./images/jira_workflow1.png)
+    ![label picture jira](./images/jira_workflow2.png)
+
+
+* There are a few necessary elements to include:
+
+* label 
+    ![label picture jira](./images/jira_workflow3.png)
+    * please makre sure that you add the cohort-resources label
+
+* Summary
+    * This is the 'at a glance' view of the issue being created. Please include the course where the issue is occurring as well as a terse description (~15 words or less) of what's going wrong.
+
+    * Example: [CR] add study guide for active record
+
+* Description
+    * Here is your chance to explain the issue as best you can in as much detail as you can muster and give us everything we need to fix the issue. A few things MUST be present:
+
+    1. a link to the place where the issue is happening or where you are adding something
+    2. a description of what you observe and why this conflicts with what you think it should be
+    priority, in your opinion (described below)
+        * To make our life even easier please provide a screenshot if applicable or even a suggestion for how the problem should be solved.
+
+* Estimating priority
+    * Please set priority in line with the following guidelines.
```

Comment:
small thing, possibly personal preference, but ending this line with a **:** (colon)?